lib/key-handler.js

```diff
@@ -4,20 +4,32 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import {canUseDOM} from 'exenv';
 
-import {KEYDOWN, KEYPRESS, KEYUP} from './constants';
-import {isInput, matchesKeyboardEvent, eventKey} from './utils';
+import {KEYDOWN, KEYPRESS, KEYUP, type KeyConstants} from './constants';
+import {type primitiveOrArray} from './types';
+import {isInput, matchesKeyboardEvent} from './utils';
 
 type Props = {|
-  +keyValue?: string,
-  +keyCode?: number,
-  +keyEventName: KEYDOWN | KEYPRESS | KEYUP,
-  +onKeyHandle?: (event: KeyboardEvent) => void,
+  +keyValue?: ?primitiveOrArray<string>,
```

Comment:
What are the `+` prefixes for?

Reply:
TIL they indicate the props are readonly, see flow docs https://flow.org/en/docs/types/interfaces/#toc-interface-property-variance-read-only-and-write-only

Reply:
Sorry, wrote the answers yesterday but there was a "pending" set to the answer. TIL - you need to submit pending comments at the top of the page :grimacing: